src/Utilities/Compiler/Options/EditorConfigOptionNames.cs

```diff
@@ -67,6 +67,13 @@ internal static partial class EditorConfigOptionNames
         /// </summary>
         public const string AdditionalStringFormattingMethods = "additional_string_formatting_methods";
 
+        /// <summary>
+        /// String option to enable heuristically detecting of additional string formatting methods for CA2241 (https://docs.microsoft.com/visualstudio/code-quality/ca2241).
```

Comment:
This exact comment can be added to the `Analyzer Configuration.md` doc section.

Reply:
I noticed that the comment is wrong, it should read 'Boolean option'.